src/network/packets_processor/thread_pool.cpp

```diff
@@ -0,0 +1,149 @@
+#include "thread_pool.hpp"
+
+namespace taraxa::network {
+
+ThreadPool::ThreadPool(size_t workers_num) : stopProcessing_(false), queue_(), mutex_(), cond_var_(), workers_() {
```

Comment:
we have another threadpool in namespace taraxa::util that is used everywhere, maybe we should have just one generic

Reply:
I know one proposed architecture is that all tasks within the node run in a priority queue more globally than just transaction packet handling.

Reply:
good idea, but not something we want to solve by this PR...